### ISSUE TITLE
Alignment attribute when converting to XML

### ISSUE THREAD

**Issue by maelle** [github/cmark#115]
:wave: here! Thanks for a great project! 

It seems that when converting to XML, the alignment attribute of table cells in table is lost (it is not lost when converting to html). Would it be possible to add support for this in the XML conversion?

Linked to https://github.com/jeroen/commonmark/issues/9

cc @jeroen

**Comment by kivikakk** [comment]
Definitely! We currently don't support custom XML formatters for extensions, like we do for HTML/LaTeX/etc.; we'll need to add this to make it possible.

Would you be interested in writing up a PR for this? I'd be happy to give pointers.

**Comment by kivikakk** [comment]
Ah, got it! I'll do this as soon as I get a chance, then 👍

**Comment by maelle** [comment]
Thank you!
